### DISCUSSION TITLE
Graph database input?

### DISCUSSION THREAD

**Discussion by juju4** [dbarzin/mercator#459] [General]
Thanks for sharing your tool.

I'm not sure but it seems tools input is manual at this point, right?
Is there interest to collect automatically from other tools, especially graph-database ones like [BloodHound](https://github.com/BloodHoundAD/BloodHound), [Starbase](https://github.com/JupiterOne/starbase), [Cartography](https://github.com/lyft/cartography)?

**Comment by dbarzin** [comment]
Inputs are manual but there is a REST API to automate it.
If you want to participate, we are open to pull requests !
